docs/cse/rules/insight-trainer.md

```diff
@@ -0,0 +1,97 @@
+---
+id: insight-trainer
+title: Improve Rules with Insight Trainer
+sidebar_label: Improve Rules with Insight Trainer
+description: The Cloud SIEM Insight Trainer dashboard recommends adjustments to rules to improve Insight generation.  
+keywords:
+  - cloud siem
+  - insight trainer
+  - rule tuning
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+[Cloud SIEM - Insight Trainer](/docs/integrations/sumo-apps/cse#insight-trainer) is a dashboard in the Enterprise Audit - Cloud SIEM app. Insight Trainer offers suggestions for making adjustments to rules, such as writing rule tuning expressions and changing severities. Implementing the recommendations causes rules to be more effective at creating high-fidelity Signals, resulting in generation of more meaningful Insights. 
+
+## About Insight Trainer
+
+When you resolve Insights, you give them [Insight resolutions](/docs/cse/administration/manage-custom-insight-resolutions/) such as: 
```

Comment:
Should this be "you're given"?

Reply:
It's good as-is. The customer actually gives a resolution to an Insight when they close it.